app/components/price-graph/price-graph.tsx

```diff
@@ -101,7 +101,9 @@ export const PriceGraphDataInjected = () => {
 
   return (
     <PriceGraph
-      prices={data.btcPriceList.slice()} // FIXME: PricePoint should be non-nullable
+      prices={data.btcPriceList
+        .filter((price) => price !== null)
+        .map((price) => price as PricePoint)} // FIXME: PricePoint should be non-nullable
```

Comment:
I think you have fixed the FIXME :)

Reply:
Maybe I could be more clear, I think the backend should make it non nullable, because right now we receive an array will some of the values possibly being null

Reply:
yup would be relevant to precise the backend should be updated here. 

a nullable to non nullable is not a breaking change afair so should be possible easily